sift/recorder/mfile.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
There is a larger question on how to handle the Python2 vs. Python3 question in Sniper. Right now, I've been leaning toward using Python2 directly everywhere until we properly move to Python3. This is because many of the scripts in Sniper require Python2 currently.